reference/apm/observability/apm.md

```diff
@@ -21,7 +21,7 @@ Metrics are another vital source of information when debugging production system
 
 ## Give Elastic APM a try [_give_elastic_apm_a_try]
 
-Use [Get started with application traces and APM](/solutions/observability/apm/get-started-fleet-managed-apm-server.md) to quickly spin up an APM deployment. Want to host everything yourself instead? See [Get started](/solutions/observability/apm/get-started.md).
+Use [Get started with application traces and APM](/reference/fleet/get-started-managed-apm-server.md) to quickly spin up an APM deployment. Want to host everything yourself instead? See [Get started](/solutions/observability/apm/get-started.md).
```

Comment:
The link text *Get started with application traces and APM* no longer matches the page heading